lib/haml_lint/ruby_extractor.rb

```diff
@@ -187,6 +187,11 @@ def add_line(code, node_or_line, discard_blanks = true)
       end
     end
 
+    def indentation_code(code, indent)
```

Comment:
With a verb, this method name could be `indent_code` (which is more in line with other methods in this project).